vanilla-framework/vanilla-framework
# Update docs/strip page content

## Done
- Added note notifications
- Updated headings
- Updated demo example strips to showcase live examples 😉 
- Check content matches copy doc [here](https://docs.google.com/document/d/1eq56B2iEauk9g4bRVEJtiO9hQ7Y6yuJP6zcLN-tLess/edit#heading=h.ifh1f0ulklnm)

## QA
- Pull code
- Run `./run serve --watch`
- Check https://docs.vanillaframework.io/en/patterns/strip
- Review updated note notifications
- Check http://0.0.0.0:8101/vanilla-framework/
- Review updated strip examples and note notifications

## Details
- Fixes https://github.com/ubuntudesign/vanilla-design/issues/342

## Screenshots
![screenshot 2019-03-05 at 16 53 41](https://user-images.githubusercontent.com/17748020/53822247-4565ed80-3f67-11e9-8c8e-c8bae4ce3f4f.png)

![screenshot 2019-03-05 at 16 57 44](https://user-images.githubusercontent.com/17748020/53822525-d341d880-3f67-11e9-8fe6-753876fe1d5d.png)
